components/brave_wallet_ui/components/extension/connected-panel/index.tsx

```diff
@@ -122,6 +123,13 @@ const ConnectedPanel = (props: Props) => {
 
   const onClickViewOnBlockExplorer = useExplorer(selectedNetwork)
 
+  const showConnectButton = React.useMemo((): boolean => {
+    if (originInfo.origin) {
+      return originInfo.origin.startsWith('chrome://')
```

Comment:
can this be an exact check for a set of whitelisted origins? i'm wondering why all of `chrome://` needs to be whitelisted.

Reply:
While addressing this issue https://github.com/brave/brave-browser/issues/20973 it was decided there is no reason to allow a user to `give/remove` a `Site Permission` to `chrome://` origins. 
cc: @bbondy @srirambv